python/BioSimSpace/FreeEnergy/_relative.py

```diff
@@ -417,7 +675,292 @@ def _analyse(self):
         return Relative.analyse(self._work_dir)
 
     @staticmethod
-    def _analyse_gromacs(work_dir=None):
+    def preprocessing_extracted_data(data):
+        """_summary_
+
+        Args:
+            data : dataframe
+            Dataframe of extracted dHdl or u_nk data.
+
+        Returns
+        -------
+
+            processed_data : dataframe
+            Dataframe of dHdl or u_nk data processed using automated equilibration
+            detection followed by statistical inefficiency.
```

Comment:
I've changed it to private for now, but after some discussion with @jmichel80 I think it might be good for the user to have more input about how their data is preprocessed. I'm not sure what the best way would be to go about this, as the preprocessing is quite hidden in the analyse part right now (as a public API? or via extra options?), but I think this would also be good to discuss more after the workshop.